include/librealsense2/hpp/rs_device.hpp

```diff
@@ -569,11 +569,48 @@ namespace rs2
         void release() override { delete this; }
     };
 
+    class device_passive_calibration : public device
```

Comment:
Can we call this `calibration_change_device`?

Reply:
Renamed to `calibration_change_notifier`

Reply:
I don't like this name: if it's a type of device, then `device` should be in the name.